doc/nrf/releases/release-notes-latest.rst

```diff
@@ -187,44 +289,138 @@ The following list summarizes the most important changes inherited from upstream
     * Unified endpoint helper macros across all USB device drivers.
     * Fixed handling of fragmented transfers on the control OUT endpoint in the Nordic Semiconductor USB Device Controller driver (usb_dc_nrfx).
     * Introduced names for threads used in USB classes, to aid debugging.
+    * The usb_enable() function, which, for some samples, was invoked
+      automatically on system boot up, now needs to be explicitly called
+      by the application in order to enable the USB subsystem. If your
+      application relies on any of the following Kconfig options, then
+      it shall also enable the USB subsystem:
+      * :option:`CONFIG_OPENTHREAD_NCP_SPINEL_ON_UART_ACM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_ECM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_EEM`
+      * :option:`CONFIG_USB_DEVICE_NETWORK_RNDIS`
+      * :option:`CONFIG_TRACING_BACKEND_USB`
+      * :option:`CONFIG_USB_UART_CONSOLE`
+    * cdc class: Do not duplicate USB transfer slots for OUT transfers.
+    * cdc_acm: re-trigger transfer for OUT endpoint on resume. 
+    * nRF: Fix for remote wake up request.
+    * hid: Allow to send data only in CONFIGURED state.
+    * Switched to using kernel stack.
+    * audio: Correct typo in USB audio DTS bindings.
 
   * Watchdog:
 
     * Updated the description of the :cpp:func:`wdt_feed` API function to reflect an additional error return code.
 
+  * IEEE 802.15.4:
```

Comment:
This should be moved up, to keep the alphabetical order.

Reply:
moved